index.html

```diff
@@ -4,6 +4,8 @@
     <link rel="stylesheet" type="text/css" href="static/styleguide.css">
     <script src="static/styleguide.js"></script>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700,300' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
```

Comment:
We are using font-awesome-sass gem version 5.12 which at the end is fontawesome 5.12.0.

Maybe we can use this other CDN:

`<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.0/css/all.min.css" crossorigin="anonymous">`

just in case something changed between those version. It looks good though, just to be safe